releasenotes/notes/fix-registerless-one-bit-displays-4deb8f7cecf3f602.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
```

Comment:
This is a v minor comment but I think it would be better to split this into 2 bullet points (one for each issue that was fixed), see code snippet example [here](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#release-notes) in the contributing guidelines

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/6942/commits/8a7c24a01ea3806e4e93c0d47ad0171101130b4c.